Client/Extensions/NSURLExtensionsMailTo.swift

```diff
@@ -26,7 +26,7 @@ public extension URL {
         let urlString = absoluteString
 
         // Extract 'to' value
-        let toStart = urlString.characters.index(urlString.startIndex, offsetBy: "mailto:".characters.count)
+        let toStart = urlString.characters.index(urlString.startIndex, offsetBy: "mailto:".count)
```

Comment:
Isn't there still another use of `characters` here?

Reply:
Yes, I'll search for more usages I missed.